### ISSUE TITLE
When on the same instance as the post, or comment, make the "View On Instance" button greyed-out/uninteractable/invisible.

### ISSUE THREAD

**Issue by K4LCIFER** [LemmyNet/lemmy-ui#2025]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

It can be confusing for a new user to click the "View On Instace" button, and to only see their page suspiciously reload with nothing change, if they are on the same instance as the post, or comment that they are viewing. To prevent this confusion from happenening, I propose that the button simply becomes uninteractable in some way wen on the same instance as the post, or comment in question.

**Comment by beedell-roke** [comment]
> https://github.com/LemmyNet/lemmy-ui/issues/2025#event-18889752213

@dessalines, why? I was confused by this as a new user for some time.

**Comment by dessalines** [comment]
PRs welcome.

**Comment by beedell-roke** [comment]
@dessalines, indeed, but that doesn't explain why you've ceased to track a defect.
